F811 redefinition of unused ''

flake8 is checking for a redefinition of a method which is generally fine, but is required by the new odoo API when we want a new and old API compatibility : 

Exemple in `account`: https://github.com/odoo/odoo/blob/8.0/addons/account/account_invoice.py#L1107-L1196

I have an example in a community module too: https://github.com/guewen/bank-statement-reconcile/blob/0b84dda3ff6f58c07511928add4a9af0eba69849/account_invoice_reference/account_invoice.py#L28-L38

```
flake8 . --config=${FLAKE8_CONFIG_DIR}/travis_run_flake8.cfg

./account_invoice_reference/account_invoice.py:36:5: F811 redefinition of unused '_ref_from_invoice' from line 28
```

Should we remove this check?

--- a/travis/cfg/travis_run_flake8.cfg
+++ b/travis/cfg/travis_run_flake8.cfg
@@ -1,3 +1,4 @@
 [flake8]
+ignore = F811
 max-line-length = 79
 exclude = __unported__,__init__.py

